### DISCUSSION TITLE
Show Keybinding in Hover Tooltips

### DISCUSSION THREAD

**Discussion by qhga** [dominiksta/wournal#3] [Ideas]
Show the corresponding keybind for the tools / functions shown in the toolbars

![image](https://github.com/dominiksta/wournal/assets/15987853/c61e93e6-fb80-43b1-88d6-adf8c7d2e16f)

For example, when hovering over the `Zoom In` icon, the tooltip should say: `Zoom In (Ctrl + '+', Ctrl + Scroll Up)`

**Comment by dominiksta** [comment]
Thanks for the suggestion! This is now implemented and coming in the next release of Wournal. (Implemented in 2f9bfde15e9d00f205153d292df85f2a88e33b90.)
